billsplit/src/utils/calculateTotalAmout.js

```diff
@@ -0,0 +1,27 @@
+
+export const calculateLogisticsTotal = bill => {
+  
+  if (bill.taxPercent.length === 0) {
+    let initial =
+      parseInt(bill.amountTotal) / parseInt(bill.numberOfPartyMembers);
+    let tipPerPerson = parseInt(bill.amountTotal) * (parseInt(bill.tipPercent) / 100);
+    let amountPerPerson = initial + tipPerPerson;
+    return amountPerPerson;
+  
+  } else if (bill.tipPercent.length === 0) {
+    let taxAmount = parseInt(bill.amountTotal) * (parseInt(bill.taxPercent) / 100);
+    let initial = parseInt(bill.amountTotal) + taxAmount
+    let amountPerPerson = (initial / (parseInt(bill.numberOfPartyMembers)))
+    return amountPerPerson
+  
+  } else if (!bill.tipPercent && !bill.taxPercent) {
+    let amountPerPerson = parseInt(bill.amountTotal) / parseInt(bill.numberOfPartyMembers);
```

Comment:
There is probably a DRY-er way to present this since the amount per person calculation is always the same. Perhaps you can separate the calculation for amount per person to a separate function that you call within your conditional logic flow here as appropriate.